cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -23,6 +24,22 @@
 ], scope="module")
 
 
+@pytest.fixture(scope="module")
+def configure_websocket(appliance):
+    """
+    Enable websocket role if it is disabled.
+
+    Currently the fixture cfme/fixtures/base.py:27,
+    disables the websocket role to avoid intrusive popups.
+    """
+    logger.info('Enabling the websocket role to allow console connections')
+    server_information = server_settings.ServerInformation(appliance)
+    server_information.enable_server_roles('websocket')
+    yield
+    logger.info('Disabling the websocket role to avoid intrusive popups')
+    server_information.disable_server_roles('websocket')
```

Comment:
**REQUIRED**
It can be done easier:
```python
from cfme.configure.configuration import server_roles_enabled
...
@pytest.fixture(scope="module")
def configure_websocket(appliance):
    with server_roles_enabled('websocket'):
        yield
```

Reply:
@quarckster Done :)